### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by zhouzhuojie**
## Description
Fix https://github.com/checkr/flagr/issues/442
The reason is that https://stackoverflow.com/a/66910240, removing `MYSQL_USER` set in the docker-compose file fixes the problem, note that `root` is automatically created.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by zhouzhuojie** [comment]
@jperichon @sesquipedalian-dev can you take a look and give a stamp?

BTW, a few PRs (https://github.com/checkr/flagr/pull/441 https://github.com/checkr/flagr/pull/439) are blocked by this, I'm also interested in getting the PR merge permission for flagr related repos, let me know if that works for you.

**Review by sesquipedalian-dev — APPROVED** [review]
Looks good, thanks for the fix

**Pull request merged**
